packages/fstar/fstar.0.9.0/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "protz@microsoft.com"
+homepage: "http://fstar-lang.org"
+license: "Apache"
+depends: [
+  "ocamlfind"
+  "batteries"
```

Comment:
Not that I know of. I think the usage of batteries was pretty standard. @nikswamy do you have any info?